mtgbp/chain.json

```diff
@@ -0,0 +1,77 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "mtgbp",
+    "status": "live",
+    "network_type": "mainnet",
+    "website": "https://mtgbp.com/",
+    "pretty_name": "mtgbp",
+    "chain_id": "mtgbp-1",
+    "bech32_prefix": "mtgbp",
+    "daemon_name": "mtgbp",
+    "node_home": "$HOME/.ssc",
+    "key_algos": [
+      "secp256k1"
+    ],
+    "slip173": "mtgbp",
+    "slip44": 1179993441,
```

Comment:
Pending PR (which need fixing)
https://github.com/satoshilabs/slips/pull/1740

Reply:
@JeremyParish69, Can you help me figure out what they are asking me to change in SLIP44 ?
The comments of the reviewer are mentioned below.
    1. Keep the list sorted
    2. Make sure the dec values match the hex values
    3. Please fix also TTNC value, which is wrong

Reply:
The slip173 entries you've just added at the bottom of the chart, as opposed to arranging it in alphabetical order.
The hexadecimal values in your slip44 entries do not match the decimal values.
Not sure about TTNC

Reply:
@JeremyParish69  Thank you for giving me proper understanding.

Reply:
I see you made changes in that PR recently but it's still not fixed.

Reply:
Still something is remaining from my side to correct ? @JeremyParish69

Reply:
Yes, your registration to slip173 and slip44 need to be accepted. You have to match the existing convension.

Reply:
Can you share with me those conventions to match?

Reply:
A maintainer of that repo left on comment on your PR that you haven't addressed. The one that stick out to me is the ordering. You aren't following ascending and alphabetical ording

Reply:
Please check. I have arranged the list in order form.

Reply:
@JeremyParish69 Please check the SLIP0044 and SLIP0172 to get **MERGED**.